modules/preview-react/text/stories/Text.stories.mdx

```diff
@@ -19,17 +19,18 @@ simplified usage of Canvas Kit type token.
 ## Installation
 
 ```sh
-yarn add @workday/canvas-kit-react
+yarn add @workday/canvas-kit-preview-react
 ```
 
 ## Text
 
 ### Usage
 
-`Text` is a basic text component to render modified text. `Text` component composes `Box` component
-that allows to use all layout props, such as `display`, `margin`, `lineHeight` etc. It also has
-their own props such as `level` and `size` that should be provided together, because they refer to
-type token level and size, and `variant` to set variant color.
+`Text` is the base component that is used to render text in the UI. It is built off of our `Box`
+component which allows the use of styles props such as `display`, `margin`, `padding` as well as the
+most common type styles such as `fontWeight`, `fontSize` amongst others. This component is an
+extension of our type tokens(insert link), therefore, props like `variant`, `level` and `size` all
```

Comment:
extension of our [type tokens](https://canvas.workdaydesign.com/tokens/type#type-styles), therefore, props like `variant`, `level` and `size` all